src/runtime_src/tools/xclbinutil/unittests/FixedKernel/fixed_kernel_add.json

```diff
@@ -5,7 +5,8 @@
         "name" : "DPU",   
         "type" : "dpu",
         "extended-data" : {
-            "functional" : "0",
+            "functional" : "DPU",
```

Comment:
Because there is EXISTING customer code.  The functional value needs to be able to support BOTH numeric and string values when it is being read in.   

Note: For writing, there is no such requirement and string values is preferred.

Reply:
add support to handle both string and numeric usecases